CONTRIBUTING.md

```diff
@@ -238,3 +238,8 @@ Snapshot management requires VM t be powered off.
 ### QEMU on MacOS
 
 You might want try to use [UTM](https://mac.getutm.app/).
+
+## Documentation
+
+If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,
+see that they are built from the repository [replicatedhq/kurl.sh][kulr-docs].
```

Comment:
do you have this target already? `kulr-docs` also it is misspelled

Reply:
Sorry, it came from the split. 
I added all links with alias but in the review was requested not to use alias. 
Fixing that now.